caasp-stack.yaml

```diff
@@ -231,8 +294,37 @@ resources:
             runcmd:
               - /usr/bin/systemctl enable ntpd
               - /usr/bin/systemctl start ntpd
+              - mkdir /etc/caasp/cpi
+              - chmod -R 0777 /etc/caasp/cpi
+
+            write_files:
+              - path: /etc/caasp/cpi/openstack.conf
+                permissions: 0777
+                content: |
+                  auth-url=$cpi_auth_url
+                  domain-name=$cpi_domain_name
+                  tenant-name=$cpi_tenant_name
+                  region=$cpi_region
+                  username=$cpi_username
+                  password=$cpi_password
+                  subnet-id=$cpi_subnet_id
+                  floating-network-id=$cpi_floating_network_id
+                  monitor-max-retries=$cpi_monitor_max_retries
+                  bs-version=$cpi_cinder_bs_version
```

Comment:
Please change it to `bs-version=$cpi_bs_version`

Reply:
done outdated